workshops/arrow.qmd

```diff
@@ -28,11 +26,17 @@ This course is for you if you:
 
 -   want to learn how to engineer your tabular data storage for more performant access and analysis with Apache Arrow
 
+This course is probably not for you if:
+
+- you already have extensive Arrow experience and want to learn about the framework in general rather than how to work with it in R
```

Comment:
```suggestion
- you already have extensive Arrow experience and want to learn about the framework in general rather than how to work with it in R
```